src/email-assets/email-templates/insights-weekly-email.html

```diff
@@ -0,0 +1,447 @@
+                                            <!-- <div id="systems">{{systems.total}}</div> -->
+                                            <div id="systems">18</div>
+                                          </a>
+                                        </td>
+                                      </tr>
+                                      <tr>
+                                        <td align="center">
+                                          <!-- <a href="https://cloud.redhat.com/insights/inventory" target="_blank">System{{systems.total|pluralize}}</a> -->
+                                          <a class="rh-masthead__metric-block-text" href="https://cloud.redhat.com/insights/inventory" target="_blank">Systems</a>
+                                        </td>
+                                      </tr>
+                                    </table>
+                                  </td>
+
+                                  <!--Graphic 3-->
+                                  <td width="33.3333%" valign="top">
+                                    <table role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%">
+                                      <tr>
+                                        <td align="center">
+                                          <a class="rh-masthead__metric-block-icon" href="https://cloud.redhat.com/insights/rules?impacting=true&reports_shown=true" target="_blank"><img src="https://raw.githubusercontent.com/RedHatInsights/frontend-assets/master/src/email-assets/img_total-hits.png" alt="total hits" width="90" height="90" />
+                                          </a>
+                                        </td>
+                                      </tr>
+                                      <tr>
+                                        <td align="center">
+                                          <a class="rh-masthead__metric-block-count" href="https://cloud.redhat.com/insights/rules?impacting=true&reports_shown=true" target="_blank">
+                                            <!-- <div id="total-hits" style="color:#ffffff;">{{rules.total}}</div> -->
+                                            <div id="total-hits">120</div>
+                                          </a>
+                                        </td>
+                                      </tr>
+                                      <tr>
+                                        <td class="rh-masthead__metric-block-text" align="center">
+            <!--                               <a href="https://cloud.redhat.com/insights/rules?impacting=true&reports_shown=true" target="_blank" style="color:#ffffff;">Total hit{{rules.total|pluralize}}</a> -->
+                                          <a href="https://cloud.redhat.com/insights/rules?impacting=true&reports_shown=true" target="_blank">Total hits</a>
```

Comment:
this url needs to be updated, the advisor url is now:

`https://cloud.redhat.com/insights/advisor/recommendations`